tests/ci_build/jenkins_tools_spark.Groovy

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/groovy
```

Comment:
If this is created by us, better to remove it

Reply:
Good findings, Done